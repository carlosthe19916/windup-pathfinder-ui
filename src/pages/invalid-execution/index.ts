export { InvalidExecution as default } from "./invalid-execution";
