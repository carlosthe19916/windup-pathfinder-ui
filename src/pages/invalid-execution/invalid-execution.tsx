import React from "react";
import {
  EmptyState,
  Title,
  EmptyStateBody,
  EmptyStateVariant,
  EmptyStateIcon,
  Bullseye,
} from "@patternfly/react-core";
import { UserNinjaIcon } from "@patternfly/react-icons";

export const InvalidExecution: React.FC = () => {
  return (
    <Bullseye>
      <EmptyState variant={EmptyStateVariant.large}>
        <EmptyStateIcon icon={UserNinjaIcon} />
        <Title headingLevel="h4" size="lg">
          The execution you are trying to load is not valid
        </Title>
        <EmptyStateBody>
          The execution folder must have a{" "}
          <strong>TitanConfiguration.properties</strong> file and{" "}
          <strong>graphsearch</strong> and <strong>titangraph</strong> folders.
        </EmptyStateBody>
      </EmptyState>
    </Bullseye>
  );
};
