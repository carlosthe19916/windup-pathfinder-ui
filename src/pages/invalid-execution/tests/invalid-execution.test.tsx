import React from "react";
import { shallow, mount } from "enzyme";
import { InvalidExecution } from "../invalid-execution";

describe("Welcome", () => {
  it("Renders without crashing", () => {
    const wrapper = shallow(<InvalidExecution />);
    expect(wrapper).toMatchSnapshot();
  });
});
