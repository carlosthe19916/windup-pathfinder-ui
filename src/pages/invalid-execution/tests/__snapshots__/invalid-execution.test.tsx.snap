// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome Renders without crashing 1`] = `
<Bullseye>
  <EmptyState
    variant="large"
  >
    <EmptyStateIcon
      icon={[Function]}
    />
    <Title
      headingLevel="h4"
      size="lg"
    >
      The execution you are trying to load is not valid
    </Title>
    <EmptyStateBody>
      The execution folder must have a
       
      <strong>
        TitanConfiguration.properties
      </strong>
       file and
       
      <strong>
        graphsearch
      </strong>
       and 
      <strong>
        titangraph
      </strong>
       folders.
    </EmptyStateBody>
  </EmptyState>
</Bullseye>
`;
